src/java.desktop/unix/native/libawt_xawt/awt/gtk2_interface.c

```diff
@@ -215,7 +215,7 @@ static GtkWidget* (*fp_gtk_label_new)(const gchar* str);
 static GtkWidget* (*fp_gtk_menu_new)();
 static GtkWidget* (*fp_gtk_menu_bar_new)();
 static GtkWidget* (*fp_gtk_menu_item_new)();
-static GtkWidget* (*fp_gtk_notebook_new)();
+static GtkWidget* (*fp_gtk_notebook_new)(void);
```

Comment:
This change is inconsistent with the surrounding style; I'd keep the original declaration, and only remove the NULL in function calls.

Reply:
I did not notice it is not required to change the prototype. OK, it works without this change.